### ISSUE TITLE
Footer a11y

### ISSUE THREAD

**Issue by zoeabryant** [redbadger/website-honestly#114]
- [x] OLD STREET is read out twice. It might make more sense to say `Old Street` and `Old Street station` on the SVG map, as the screenreader doesn't know one of the `Old Street` texts indicates a station
  **conclusion:** convert text to svg
- [x] give context to contact us details - After the navigation list it launches into reading the email address, and it's a confusing switch. Can we prefix this text with 'email us on', 'call us' etc? We could visually hide this if needed. 
  How it is read:
  `...List endLink hello at red dash badger.com Link plus forty-four left paren zero right paren twenty three thousand five hundred sixty-seven five hundred fifty-five List of eight items bullet Link Github...`
  **conclusion:** prefix it, but hide it visually
- [x] svgs shouldn't have metadata, it should be on the object. SVG metadata should be torn out
- [x] Look up most used screenreaders, get information on what we should be testing with.

http://webaim.org/projects/screenreadersurvey6/

**Comment by zoeabryant** [comment]
fixed with #113
